azure-pipelines/builds/ci.yml

```diff
@@ -19,20 +19,44 @@ stages:
 - stage: build
   displayName: Build
   jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
-    parameters:
-      enablePublishUsingPipelines: true
-      enablePublishBuildArtifacts: true
-      enablePublishBuildAssets: true
-      publishAssetsImmediately: true
-      artifacts:
-        publish:
-          artifacts: true
-          manifests: true
-      enableSourceBuild: true
-  - template: /azure-pipelines/builds/generatescript-tests.yml
-    parameters:
-      buildConfiguration: Debug
+  - job: Linux
+    pool:
+      vmImage: ubuntu-latest
```

Comment:
I think it would be best to utilize the same docker image that arcade runs the sb legs in.  This will help provide isolation from changing dependencies.

Reply:
Addressed in https://github.com/dotnet/source-build-reference-packages/pull/824/commits/5f4f2f7d80af842110ae1fccd2145807c0d9f369